lib/data/data_source/remote/database_remote_source_impl.dart

```diff
@@ -96,4 +99,30 @@ class DatabaseRemoteSourceImpl implements DatabaseRemoteSource {
       throw ApiException(Errors.somethingWentWrong);
     }
   }
+  @override
+  Future<Success> updateUserProfile(UserProfileDto dto) async {
+    try {
+      final userId = firebaseAuth.currentUser?.uid;
```

Comment:
You can create a getter for the userId to not write this all the time

Reply:
You created the getter but you don't use it everywhere 😛